From: Shengjing Zhu <zhsj@debian.org>
Date: Tue, 22 Feb 2022 00:39:13 +0800
Subject: Skip failed tests

on buildd:

RUN   TestSocketDiagTCPInfo
socket_test.go:68: no such file or directory
FAIL: TestSocketDiagTCPInfo (0.00s)

RUN   TestHandleCreateClose
handle_test.go:25: protocol not supported
FAIL: TestHandleCreateClose (0.01s)

RUN   TestHandleTimeout
handle_test.go:120: protocol not supported
FAIL: TestHandleTimeout (0.00s)

RUN   TestHandleReceiveBuffer
handle_test.go:138: protocol not supported
FAIL: TestHandleReceiveBuffer (0.00s)

Forwarded: not-needed
---
 handle_test.go | 2 +-
 socket_test.go | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/handle_test.go b/handle_test.go
index ac627ba..b6ef7a6 100644
--- a/handle_test.go
+++ b/handle_test.go
@@ -1,4 +1,4 @@
-// +build linux
+// +build ignore
 
 package netlink
 
diff --git a/socket_test.go b/socket_test.go
index f21520f..2bcd540 100644
--- a/socket_test.go
+++ b/socket_test.go
@@ -59,6 +59,7 @@ func TestSocketGet(t *testing.T) {
 }
 
 func TestSocketDiagTCPInfo(t *testing.T) {
+	t.Skip("broken on buildd")
 	Family4 := uint8(syscall.AF_INET)
 	Family6 := uint8(syscall.AF_INET6)
 	families := []uint8{Family4, Family6}
